01-2024-schedule/week-3.md

```diff
@@ -0,0 +1,39 @@
+| **Friday** | <ul><li>[Transactions fees Structure](https://learnmeabitcoin.com/technical/transaction-fee)</li> <li> [How Wallets Can Handle Transaction Fees](https://bramcohen.medium.com/how-wallets-can-handle-transaction-fees-ff5d020d14fb). </li>  <li> [Fee Estimation by Wallets](https://btctranscripts.com/scalingbitcoin/montreal-2015/transaction-fee-estimation/). </li> <li> [Out of Band](https://bitcoinops.org/en/topics/out-of-band-fees/). </li> <li> [An Intro to Bitcoin Core Fee Estimation](https://johnnewbery.com/an-intro-to-bitcoin-core-fee-estimation/).</li>  <li> [Challenges of Fee Estimation](https://hackmd.io/_vQeKNLeSPeh0g-uhbA5_A). </li> <li> [Coinselection](https://bitcoinops.org/en/topics/coin-selection/).  </li> <li> [Transaction Pinning](https://bitcoinops.org/en/topics/transaction-pinning/). </li> <li> [Replace By Fee (RBF)](https://bitcoinops.org/en/topics/replace-by-fee/). </li> <li> [Replace By Fee Fee BIP](https://github.com/bitcoin/bips/blob/master/bip-0125.mediawiki) </li></ul> | Understand: <ul> <li>How fees are estimated for transactions. </li> <li>The difference between absolute and variable fee rate. </li> <li>The difference betwwen RBF and CPFP for fee bumping transactions. </li> <li> Pinning attacks. </li> |  <ul> <li> Create transactions on regtest with 2s/vb, 10s/vb, 15s/vb, and 10,000 sats absolute fee. </li> <li> ***Bonus***: Create a 2-generation chain of transactions with an ancestor score of 20s/vb. </li></ul> |
+
+*Note: If you complete the day's readings ahead of schedule, consider progressing to the next day's material, as repetition enhances knowledge absorption.*
+
+## Deliverables
+
+- Complete the Block constructor code challenge.
+- Conduct the mining exercise: 51% attacks, mine a block, and reorg the chain with a double spend on regtest.
+- Bonus Exercise: Use your block construction algorithm on the live mempool to create a block template. Compare it with getblocktemplate RPC. Simulate using regtest.
+- Run the Tiny Bitcoin peer and provide an explanation of the boot process.
+- Create transactions on regtest as per the specified exercise.
+
+
+**Bonus Exercise**
```

Comment:
I think it would have been better if the Bonus exercises were under a single subheading:
- Bonus Exercises:
  - .....
  - .... etc